vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/dataview/ComboBoxListDataView.java

```diff
@@ -119,4 +120,66 @@ public Registration addItemCountChangeListener(
             ComponentEventListener<ItemCountChangeEvent<?>> listener) {
         return super.addItemCountChangeListener(listener);
     }
+
+    /**
+     * Adds a filter to be applied to all queries. The filter will be used in
+     * addition to any filter that has been set or added previously. The filter
+     * is applied on data view and is different from the filter string that can
```

Comment:
This description won't help users to understand the difference IMO. Maybe extend it something like this:
`Adds a filter to be applied to all queries. The filter will be used in addition to any filter that has been set or added previously through {@link #setFilter} or {@link #addFilter}. This filter is applied to data set permanently until it's changed through {@link #setFilter} or {@link #removeFilters}, in contrary with the client-side filter that can be typed in by user and does not modify the data set on server-side, but only defines which items are shown for a single request and erases on drop down close.

Reply:
Done.